scrapi/migrations.py

```diff
@@ -100,6 +151,7 @@ def next_page_source_partition(query, page):
 
 documents_old = ModelIteratorFactory(DocumentModelOld, next_page_old)
 documents = ModelIteratorFactory(DocumentModel, next_page_source_partition, default_args=registry.keys())
+postgres_documents = Document.objects.all()
```

Comment:
Which is where the idea of the CANONICAL_PROCESSOR came from, actually. That way when doing migrations we'd read out of the correct database.